.github/workflows/cli-integration.yml

```diff
@@ -19,7 +19,9 @@ jobs:
 
       - name: Build Docker image
         run: |
-          docker login -u "$DOCKER_LOGIN" --password-stdin <<<"$DOCKER_TOKEN"
+          [ -n "$DOCKER_LOGIN$DOCKER_TOKEN" ] \
+          && docker login -u "$DOCKER_LOGIN" --password-stdin <<<"$DOCKER_TOKEN" \
+          || echo "Not login-in to docker since no credentials were provided, might hit limits below"
```

Comment:
This echo will run if `docker login` fails.  It would be better to use an `if` block.

Reply:
indeed! thanks... alternative is to revert condition here to `-z` and doing echo on `&&`... but indeed if/then/else is more explicit, will do